doc/users/next_whats_new/axline_setters.rst

```diff
@@ -0,0 +1,10 @@
+Axline setters
+--------------
+``Axes.axline`` now supports setter methods for its *xy1*, *xy2* and *slope*
+attributes:
+
+.. code-block:: python
+
+    ax1.set_xy1(.2, .3)
+    ax1.set_slope(2.4)
+    ax2.set_xy2(.1, .6)
```

Comment:
This naming is confusing as we usually use `ax` for Axes, not lines.

Reply:
In this new commit:

- getters have been added for `xy1`, `xy2` and `slope`
- `_Axline` is made public
- new methods are added to the lines.pyi stub
- getters are added to the pytest (also changed `ax` to `line`)
- next_whatsnew is updated